src/components/Sidebar/index.jsx

```diff
@@ -70,59 +70,59 @@ const Sidebar = ({
     {
       title: 'Learner Progress Report',
       to: `${baseUrl}/admin/${ROUTE_NAMES.learners}`,
-      icon: <Icon src={TrendingUp} fixedWidth />,
```

Comment:
[context] Removing the `fixedWidth` prop resolves a prop type warning since these icons were migrated away from font-awesome.